dokka-subprojects/plugin-base/src/main/kotlin/org/jetbrains/dokka/base/renderers/html/HtmlRenderer.kt

```diff
@@ -315,12 +318,12 @@ public open class HtmlRenderer(
     ): List<Pair<DisplaySourceSet, String>> {
         var counter = 0
         return nodes.toList().map { (sourceSet, elements) ->
-            val htmlContent = createHTML(prettyPrint = false).prepareForTemplates().div {
+            val htmlContent = createSmallHTML(prettyPrint = false).prepareForTemplates().div {
                 elements.forEach {
                     buildContentNode(it, pageContext, sourceSet)
                 }
             }.stripDiv()
-            sourceSet to createHTML(prettyPrint = false).prepareForTemplates()
```

Comment:
Have you considered an optimal capacity? 
In this example, It always takes more than the default `16`.

Reply:
Yeah, I was thinking about it, but failed to think of a good default... Will it be 128 or 1024? It's hard to calculate which value will be best.

But I agree, that probably it would be nice to initialise it with bigger value. Do you have anything in mind?
I can try to check different values if needed

Reply:
> Do you have anything in mind?

I think you can find it out on stdlib.
E.g. for 128 vs 1024  - choose one with the minimal memory allocated in a profiler.

Reply:
I've tried several values (default, 128, 256, 512, 1024), the difference in memory allocations is ±3% from best to worst. With 1024 it started to behave even worse, though the diff is minimal and not always the same.
I've decided to stick with 256.